CHANGELOG.md

```diff
@@ -10,7 +10,9 @@ _None_
 
 ### New Features
 
-_None_
+* Added `config generate-xcfilelist` subcommand to generated input and output `xcfilelists` based on your configuration file. This should be used in an Xcode build step that executes `swiftgen config run`.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/dc8c7910f181388fa82098adb2e08765e00e778a/CHANGELOG.md#L13"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>